lib/pure/terminal.nim

```diff
@@ -677,3 +731,14 @@ when not defined(testing) and isMainModule:
   stdout.setForeGroundColor(fgBlue)
   stdout.writeLine("ordinary text")
   stdout.resetAttributes()
+
+when defined(windows):
+  var mode: DWORD = 0
+  discard getConsoleMode(getStdHandle(STD_OUTPUT_HANDLE), addr(mode))
+  mode = mode or ENABLE_VIRTUAL_TERMINAL_PROCESSING
```

Comment:
This is a breaking change ... Previously no escape sequences were interpreted.

Reply:
Maybe to add (enable|disable|isSupported)TrueColor procs?

Reply:
This is a breaking change, please don't do this at initialization but only if requested via ``enableTrueColor``.